client/src/test/java/com/paypal/selion/testcomponents/BasicPageImplTest.java

```diff
@@ -57,16 +58,26 @@ public void testLoadObjectMapFromMap() throws InterruptedException, IOException
 
     @Test(groups = { "functional" })
     @WebTest
-    public void testWebPageIsValidated() throws InterruptedException, IOException {
+    public void testWaitUntilPageIsValidated() throws InterruptedException, IOException {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/avnguyen/SeLion/file/1215908312/issues/source?bid=2928299&fileBranchId=2929577#l61)